core/templates/dev/head/services/stateful/FocusManagerService.js

```diff
@@ -42,6 +42,9 @@ oppia.factory('FocusManagerService', [
           _nextLabelToFocusOn = null;
         });
       },
+      setFocusLabel: function(label) {
+        _nextLabelToFocusOn = label;
+      },
```

Comment:
Hey @seanlip, is it okay to have a function in a service, that is only used while testing ? Or would it be better to make a variable in the spec file for `_nextLabelToFocusOn` using which we can initialise the `FocusManagerService`?

Reply:
Good question. I took a look at how this is being used and I would suggest neither of the approaches, actually. Having setFocusLabel() here may cause devs to use it when they should really use setFocus(); having it in the tests will lead to a skew between the `_nextLabelToFocusOn` in the service and the one in the test.

I don't understand why this new method is necessary, though. It's only called once in the tests. Why can't that particular test just call setFocus() instead (maybe with a timeout after that)?

Reply:
One of the branches in setFocus required _nextLabelToFocusOn to be true, so to test that I created this function as I couldn't find another way to set a value to that variable. Even in setFocus, it is reset to null soon after it was set.

Reply:
Ah. Then why not call setFocus() twice with different labels, before flushing the timeout and seeing where the rootscope broadcasts to?

Reply:
Ok, will try that.

Reply:
Done